contracts/english_auction_rev1.1.scilla

```diff
@@ -924,16 +954,34 @@ transition End(
         (* 2. the seller can withdraw the payment tokens. *)
         (* 3. the royalty recipient can withdraw the royalty amount. *)
         (* 4. the service fee recipient can withdraw the service fee. *)
+        (* 5. the brand recipient can withdraw the commission fee, if any *)
 
         (* Calculate royalty amount *)
         royalty_amount = portion sale_price royalty_fee_bps;
 
         (* Calculate service fee *)
         service_fee = portion sale_price svc_fee_bps;
 
+        (* Calculate commission fee *)
+        collections_contract <- collection_address;
```

Comment:
@sam-zilliqa need to check if collection_address is valid (assuming we allow collection_address to be zero address as my previous comment)